Front-End/challenge1/index.html

```diff
@@ -9,7 +9,7 @@
 <body>
 	<h1>The current count is: <span id="counter">0</span></h1>
 
-	<button id="increase">Increase</button>
+	<button id= "increase" >Increase</button>
```

Comment:
it seems this was unintentional modified

Reply:
Fixed